pages/utilites/build_database.md

```diff
@@ -44,6 +44,12 @@ What occurred:
 
 **NOTE:** OrthoMCL orthogroup assignment hinges on integrity of ortholog choices in the starting ortholog files provided. If paralogs are unknowingly present in the provided ortholog alignments the paralogs will likely be prioritized by the fisher algorithm. To investigate the level of paralogy of genes in a custom database, we strongly recommend users re-add all taxa in their custom database using the main workflow of PhyloFisher. After an initial run through the main PhyloFisher workflow that includes manual curation, `build_dataset.py` will update profile HMMs, and blast databases to promote highest level of accuracy by the fisher algorithm in subsequent runs.
 
+`build_dabase.py --rename`:
+- The `--rename` flag allows users to rename taxa in the database. 
+- `--rename` requires a tab-delimted file (.tsv) containing the Old Unique ID, New Unique ID, and New Full Name (Table 7).
+- The script will back up the database and then do a full rebuild of the database which includes re-aligning, re-trimming, and rebuilding profile HMMs using the new name provided for each taxon in the input file.
```

Comment:
Sorry I messed up earlier. No trimming occurs, and a diamond blast database is built. So re-aligning, rebuilding profile HMMs, and rebuilding of the blast database with the new taxon name(s) provided in the input file.